gemfire-sink/src/test/java/org/springframework/cloud/stream/module/gemfire/GemfireSinkIntegrationTests.java

```diff
@@ -100,8 +100,6 @@ public boolean waiting() {
 	}
 
 	@Test
-	@Ignore("Need to spawn an embedded gemfire cacahe server.  " +
```

Comment:
Well, we haven't caught the `NPE` before because this was an only test-case for `GemfireSink` and it was ignored...
:cry: